composer.json

```diff
@@ -11,18 +11,23 @@
         }
     ],
     "require": {
-        "php": ">=5.3.2",
+        "php": "^5.5 || ^7.0",
         "hackzilla/password-generator": "~1.0",
-        "symfony/framework-bundle": "~2.3|~3.0",
-        "symfony/form": "~2.3|~3.0"
+        "symfony/framework-bundle": "~2.7|~3.0",
```

Comment:
The new recommended composer syntax is `^2.7 || ^3.0`.

You should take profit to update all.